apps/rush-lib/src/data/RushHooks.ts

```diff
@@ -0,0 +1,20 @@
+import { IRushHooksJson } from './RushConfiguration';
+import Utilities from '../utilities/Utilities';
+
+export default class RushHooks {
+  private _postCommandHooks: string[];
+
+  public constructor(rushHooksJson: IRushHooksJson) {
+    this._postCommandHooks = rushHooksJson.postCommand;
+  }
+
+  public get postCommandHooks(): string[] {
+    return this._postCommandHooks;
+  }
+
+  public onPostCommand(): void {
+    this._postCommandHooks.forEach((script) => {
+      Utilities.executeShellCommand(script);
+    });
```

Comment:
Agreed

---
In reply to: [116884851](https://github.com/Microsoft/web-build-tools/pull/196#discussion_r116884851) [](ancestors = 116884851)